libs/dh/metering-point/feature-overview/src/lib/secondary-master-data/dh-secondary-master-data.component.html

```diff
@@ -0,0 +1,266 @@
+<!--
+@license
+Copyright 2021 Energinet DataHub A/S
+
+Licensed under the Apache License, Version 2.0 (the "License2");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<ng-container *transloco="let t; read: 'meteringPoint.secondaryMasterData'">
+  <watt-expansion
+    [openLabel]="t('expansion.openLabel')"
+    [closeLabel]="t('expansion.closeLabel')"
+  >
+    <div class="smd__container">
+      <p class="watt-text-l">
+        {{ t('technicalSpecificationTitle') }}
+      </p>
+      <dl>
+        <div>
+          <dt class="watt-label">
+            {{ t('netSettlementGroup') }}
+          </dt>
+          <dd class="watt-text-m">
+            <ng-container
+              *ngIf="secondaryMasterData?.netSettlementGroup; else emptyField"
+            >
+              {{ secondaryMasterData?.netSettlementGroup }}
+            </ng-container>
+          </dd>
+        </div>
+        <div>
+          <dt class="watt-label">
+            {{ t('disconnectionType') }}
+          </dt>
+          <dd class="watt-text-m" data-testid="disconnectionType">
+            <ng-container
+              *ngIf="secondaryMasterData?.disconnectionType; else emptyField"
+            >
+              {{ translationKeys?.disconnectionType | transloco }}
+            </ng-container>
+          </dd>
+        </div>
+        <div>
+          <dt class="watt-label">
+            {{ t('productionObligation') }}
+          </dt>
+          <dd class="watt-text-m">
+            <ng-container
+              *ngIf="
+                (secondaryMasterData?.productionObligation | typeof) ===
+                  'boolean';
```

Comment:
suggestion: Let's move this logic to a getter property inside the component instead of creating a dedicated pipe. The pipe solution is valid but the getter is a simpler solution that works better in this case.